src/js/letters/containers/LetterList.jsx

```diff
@@ -66,7 +71,7 @@ export class LetterList extends React.Component {
     }
 
     return (
-      <div className="step-content">
+      <div className="step-content" aria-live="polite">
```

Comment:
For some reason, this works great on the first edit, but I had trouble getting the address to read back on subsequent edits. One 'hack' I was told is that sometimes `aria-live` can be spotty and needs to be combined with an `aria-relevant` attribute. I tried all the relevant aria-relevant values and from [MDN](https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/ARIA_Techniques/Using_the_aria-relevant_attribute#Values) and `aria-relevant=additions` seemed to work the best, including on subsequent edits. I'd love to hear what you think if once you're able to compare running the page with & without that extra aria attribute.

Reply:
I tried recreating this issue and wasn't able to, the address is always read back for me when edited. Just want to clarify that you're referring to an issue within `AddressSection`, as this comment is on `LetterList`. Want to make sure I'm not missing something!

Reply:
Yep, you're not missing anything, comment is just in the wrong place.